core/trino-main/src/test/java/io/trino/block/BlockAssertions.java

```diff
@@ -549,8 +549,13 @@ public static Block createLongSequenceBlock(int start, int end)
     public static Block createLongDictionaryBlock(int start, int length)
```

Comment:
Could you add numbers as part of this commit compared to non-dictionary blocks?

Reply:
Do you mean benchmark  `DICTIONARY_HASH_BIGINT_20_PERCENT` against the same blocks but with hash block being direct `LongArrayBlock` instead `DictionaryBlock`?

Reply:
Yeah. Against same blocks but without dictionaries. But it's a nit, you can skip it